pkg/managed/services_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package managed
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/verrazzano/verrazzano-operator/pkg/monitoring"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/verrazzano/verrazzano-operator/pkg/constants"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutil"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// TestCreateServicesAppBinding model/binding is for application binding - no service is created.
+func TestCreateServicesAppBinding(t *testing.T) {
```

Comment:
is services.go just for vmi?  If so maybe it should be renamed to vmiservice.go or something.